docker-compose.yml

```diff
@@ -50,4 +50,5 @@ services:
       DATABASE_URL: postgres://postgres:postgres@postgres:5432/jira-dev
       SQS_BACKFILL_QUEUE_URL: http://localstack:4566/000000000000/backfill
       SQS_PUSH_QUEUE_URL: http://localstack:4566/000000000000/push
+      SQS_DISCOVERY_QUEUE_URL: http://localstack:4566/000000000000/discovery
```

Comment:
Adding this as it broke in my local `docker-compose up`

Reply:
Thanks!